database_cleanup/models/purge_tables.py

```diff
@@ -64,75 +44,62 @@ def purge(self, cr, uid, ids, context=None):
                 AND confrelid::regclass = '%s'::regclass;
                 """ % line.name)
```

Comment:
Ok I got it
Then we need add a custom method to raise a sql-injection in `line.name`
Because the field is filled from UI this have too much risk.